callstack/react-native-paper
# docs: add support for link to types

### Summary

PR improves documentation, by adding an option to link types with their definitions e.g. using an external resources.



### Test plan

0. Open docs
1. Go to the `Button` component
2. Scroll to the `icon` prop
3. Press `IconSource` 
4. New page with `Icon` should be opened
